antora-playbook.yml

```diff
@@ -6,7 +6,7 @@ content:
   # Test one component with each doc type being a module in it
   - url: https://github.com/OpenLiberty/docs.git
     # start_path: .
-    branches: [v*.0.0.*, !v*.0.0.*-staging, draft]
+    branches: [v*.0.0.*, `!v*.0.0.*-staging`, draft]
```

Comment:
this looks like a backtick and not single quote?
aka, shouldn't it be '!v*.0.0.*-staging' and not `!v*.0.0.*-staging`